docs/mesos/UsageGuideDockerContainerizer.md

```diff
@@ -8,165 +8,105 @@
 
 # Calico-Mesos Usage Guide with the Docker Containerizer
 
-The following instructions outline the steps required to enable and manage 
-Calico networked containers launched from Marathon using the Mesos Docker
-containerizer.  This guide covers:
--  Installing and configuring Calico
--  Creating a Docker network and managing network policy
--  Launching a container
+This guide shows how to use Marathon to start
+Calico networked containers using the Docker
+Containerizer in Mesos.  This guide covers:
+-  Creating and configuring a Docker Network with Calico
+-  Launching a Docker Task in Marathon on the created Calico Network
 
 ## Prerequisites
+This guide assumes a running Mesos Cluster that meets the following specifications:
 
-You can easily configure a mesos cluster by running through
-the [Calico Mesos Vagrant guide](./Vagrant.md). If you do this, you can skip
-directly to [Launching Containers](#launching-containers).
-
-To utilize the Docker libnetwork plugin feature of Docker, it is necessary to 
-run with a Docker version 1.9 or higher, and to configure Docker to use a
-cluster store.  
-
-#### Required Machines 
-
-The cluster must contain the following machines:
-
-- Mesos Master
-- Mesos Agent(s)
-	- Docker 1.9+ (see below)
-
-#### Required Services
-
-You will also need to run the following services somewhere in your cluster:
-
+- etcd accessible at `$ETCD_LISTEN_URL`
```

Comment:
I've switched it to `$ETCD_ADVERTISE_CLIENT_URLS`, which in the [etcd configuration docs](https://coreos.com/etcd/docs/latest/configuration.html) shows the URL etcd is accessible at. Do you think more explanation is necessary, or people will just get it?